src/main/java/com/cloudant/client/org/lightcouch/internal/URIBuilder.java

```diff
@@ -38,17 +38,26 @@
     private final StringBuilder query = new StringBuilder();
     /* key=value params */
     private final Params qParams = new Params();
+    private String userInfo = "";
 
     public static URIBuilder buildUri() {
         return new URIBuilder();
     }
 
     public static URIBuilder buildUri(URI uri) {
-        URIBuilder builder = URIBuilder.buildUri().scheme(uri.getScheme())
-                .host(uri.getHost()).port(uri.getPort()).path(uri.getPath());
+        URIBuilder builder;
+        if(uri.getUserInfo() != null && !uri.getUserInfo().isEmpty()) {
```

Comment:
I'm really confused as to why the user info stuff appears in so many places. I'm pretty sure this is going to go away when the `CookieInterceptor` is enabled though right?